packages/peregrine/lib/Toasts/useToasts.js

```diff
@@ -113,7 +113,7 @@ export const useToasts = () => {
             const {
                 dismissable,
                 message,
-                timeout,
+                timeout = DEFAULT_TIMEOUT,
```

Comment:
We specifically did not do this because passing a timeout of `0` or `false` is allowable, and actually means "do not time out".  We actually do use the default timeout later in the file:

https://github.com/magento/pwa-studio/blob/9edff927cfc5488fc74504b02a5a8d1097a2f0aa/packages/peregrine/lib/Toasts/useToasts.js#L149-L158

If you pass a `timeout` of `0` or `false`, it never times out (meaning the user has to close it manually).
If you pass a `timeout` of a number > 0, it uses times out after that.
If you do not pass a `timeout`, it times out after the `DEFAULT_TIMEOUT`.

Reply:
> We specifically did not do this because passing a timeout of 0 or false is allowable

Sure, but application of the default param here only happens if `props.timeout` is [`undefined`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters). Seems like the right change to make to me, unless I'm missing something else.

Reply:
My DevEx case was not wanting to pass any props that just reiterate what the logic determines to be default behavior. So `{ message, type }` is better than having to remember `{ message, timeout, type }` or `{ dismissable, message, type }` every time.

Reply:
> Sure, but application of the default param here only happens if props.timeout is undefined.

Huh, for some reason I thought it would not apply but I think I'm confusing `defaultProps` with destructured defaults.